src/assets/javascripts/_/index.ts

```diff
@@ -93,6 +97,8 @@ export interface Config {
   base: string                         /* Base URL */
   features: Flag[]                     /* Feature flags */
   translations: Translations           /* Translations */
+  search: string                       /* Search worker URL */
```

Comment:
we don't use these bug I guess they could be kept.  On the other hand it might be better to exclude them so that we catch at compile time any attempt to use them

Reply:
I'm ok with excluding them. If someone reports a problem, we can advise against using them or re-assess.